document/api/src/main/java/io/camunda/document/api/DocumentMetadataModel.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.document.api;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+public record DocumentMetadataModel(
+    String contentType,
+    String fileName,
+    ZonedDateTime expiresAt,
+    long size,
```

Comment:
Lets go for a `Long` in case its optional? WDYT?

Reply:
Ok, in general I would recommend to use the object and not the primitive type for this.